File: src/entity/ticket.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity;

/**
 *
 * @author firas
 */
public class ticket {
     private int numTicket; 
     
    private int nbMaxT;
    private int nbTDemande;
     private int idRes;
    private float PrixF;
    private int id_event;
    private ticketo numTicketO;

    public ticket() {
    }

    public ticket(int numTicket, int nbMaxT, int nbTDemande, int idRes, float PrixF, int id_event, ticketo numTicketO) {
        this.numTicket = numTicket;
        this.nbMaxT = nbMaxT;
        this.nbTDemande = nbTDemande;
        this.idRes = idRes;
        this.PrixF = PrixF;
        this.id_event = id_event;
        this.numTicketO = numTicketO;
    }

    public int getNumTicket() {
        return numTicket;
    }

    public void setNumTicket(int numTicket) {
        this.numTicket = numTicket;
    }

    public int getNbMaxT() {
        return nbMaxT;
    }

    public void setNbMaxT(int nbMaxT) {
        this.nbMaxT = nbMaxT;
    }

    public int getNbTDemande() {
        return nbTDemande;
    }

    public void setNbTDemande(int nbTDemande) {
        this.nbTDemande = nbTDemande;
    }

    public int getIdRes() {
        return idRes;
    }

    public void setIdRes(int idRes) {
        this.idRes = idRes;
    }

    public float getPrixF() {
        return PrixF;
    }

    public void setPrixF(float PrixF) {
        this.PrixF = PrixF;
    }

    public int getId_event() {
        return id_event;
    }

    public void setId_event(int id_event) {
        this.id_event = id_event;
    }

    public ticketo getNumTicketO() {
        return numTicketO;
    }

    public void setNumTicketO(ticketo numTicketO) {
        this.numTicketO = numTicketO;
    }

    @Override
    public String toString() {
        return "ticket{" + "numTicket=" + numTicket + ", nbMaxT=" + nbMaxT + ", nbTDemande=" + nbTDemande + ", idRes=" + idRes + ", PrixF=" + PrixF + ", id_event=" + id_event + ", numTicketO=" + numTicketO + '}';
    }

    

    
   
    

    
}

File: Downloads/integration web/src/Entity/Ticket.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Ticket
 *
 * @ORM\Table(name="ticket", indexes={@ORM\Index(name="idEvent", columns={"idEvent"}), @ORM\Index(name="numTicketO", columns={"numTicketO"}), @ORM\Index(name="idRes", columns={"idRes"})})
 * @ORM\Entity
 */
class Ticket
{
    /**
     * @var int
     *
     * @ORM\Column(name="numTicket", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numticket;

    /**
     * @var int
     * @Assert\NotNull(message="Indiquer le nombre")
     * @Assert\Positive(message="Au moins une ticket")
     * @ORM\Column(name="nbMaxT", type="integer", nullable=false)
     */
    private $nbmaxt;

    /**
     * @var int
     * @Assert\NotNull(message="Indiquer le nombre")
     * @Assert\Positive(message="Au moins une ticket")
     * @ORM\Column(name="nbTDemande", type="integer", nullable=false)
     */
    private $nbtdemande=0;

    /**
     * @var float
     * @Assert\NotNull(message="Indiquer un prix")
     * @Assert\Positive(message="Prix non négatif")
     * @ORM\Column(name="PrixF", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixf;

    /**
     * @var \Event
     *
     * @ORM\ManyToOne(targetEntity="Event")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="idEvent", referencedColumnName="IdEvent")
     * })
     */
    private $idevent;

    /**
     * @var \Ticketo
     *
     * @ORM\ManyToOne(targetEntity="Ticketo")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="numTicketO", referencedColumnName="numTicketO")
     * })
     */
    private $numticketo;

    /**
     * @var \Reservation
     *
     * @ORM\ManyToOne(targetEntity="Reservation",cascade={"persist"})
     *   @ORM\JoinColumn(name="idRes", referencedColumnName="IdRes")
     */
    private $idres;

    public function getNumticket(): ?int
    {
        return $this->numticket;
    }

    public function getNbmaxt(): ?int
    {
        return $this->nbmaxt;
    }

    public function setNbmaxt(int $nbmaxt): self
    {
        $this->nbmaxt = $nbmaxt;

        return $this;
    }

    public function getNbtdemande(): ?int
    {
        return $this->nbtdemande;
    }

    public function setNbtdemande(int $nbtdemande): self
    {
        $this->nbtdemande = $nbtdemande;

        return $this;
    }

    public function getPrixf(): ?float
    {
        return $this->prixf;
    }

    public function setPrixf(float $prixf): self
    {
        $this->prixf = $prixf;

        return $this;
    }

    public function getIdevent(): ?Event
    {
        return $this->idevent;
    }

    public function setIdevent(?Event $idevent): self
    {
        $this->idevent = $idevent;

        return $this;
    }

    public function getNumticketo(): ?Ticketo
    {
        return $this->numticketo;
    }

    public function setNumticketo(?Ticketo $numticketo): self
    {
        $this->numticketo = $numticketo;

        return $this;
    }

    public function getIdres(): ?Reservation
    {
        return $this->idres;
    }

    public function setIdres(?Reservation $idres): self
    {
        $this->idres = $idres;

        return $this;
    }


}
